packages/adaptive-ui/src/modules/selector.ts

```diff
@@ -1,44 +1,58 @@
-import type { StateSelector, StyleModuleEvaluateParameters } from "./types.js";
+import { InteractiveState, InteractiveValues } from "../types.js";
+import type { StyleModuleEvaluateParameters } from "./types.js";
 
 const HOST_CONTEXT = ":host";
 
 const defaultContext = HOST_CONTEXT;
 
 /**
- * Creates a single css selector for the provided `params` and optional `state`.
+ * Default css selectors for an interactive element's states.
+ */
+const DefaultInteractiveSelectors: InteractiveValues<string> = {
+    rest: "",
+    hover: ":hover",
+    active: ":active",
+    focus: ":focus-visible",
+    disabled: "",
```

Comment:
should disabled have a default?

Reply:
I'm not sure. There isn't a great standard here since :disabled  doesn't always work and the article isn't always there. I left it empty because of the helper functions. I made it opt-in.

Reply:
Per our chat, I'll look at cleaning up the internals of selector generation separately in the future.